tests/test_bokky_poo_bahs_date-time-library.py

```diff
@@ -62,6 +81,28 @@ def test_timestamp_from_date_automated(date_time_library, start_date, end_date):
         current_date += relativedelta(days=+1)
 
 
+@given(
+    timestamp=strategy(
+        "uint256",
+        min_value=get_timestamp_from_date(2020, 1, 1),
+        max_value=get_timestamp_from_date(2050, 1, 1),
+    )
+)
+@settings(max_examples=500, verbosity=Verbosity.quiet)
+def test_property_based_timestamp_from_date(date_time_library, timestamp):
```

Comment:
I think we should cover `addMonths` with parameter-based tests. Or, it seems we can cover it even fully, because actual values of the timestamp argument of `addMonths` are always the first seconds of a month.